collection-scripts/targeted_crs

```diff
@@ -119,12 +119,21 @@ if [ ! -z "${vm_resources}" ]; then
 
         # Gather VMs PersistentVolumeClaims
         for pvc_name in $(echo $vm_data | jq -r '.spec.template.spec.volumes[] .persistentVolumeClaim.claimName'); do
-          # In order to gather the scratch PVCs in warm migrations
-          for pvc in $(/usr/bin/oc get pvc -n ${target_ns} -o json | jq -c '.items | .[]'); do
-            if [ $(echo $pvc | jq -r '.spec.volumeName') != $pvc_name ]; then
+          pvcs=$(/usr/bin/oc get pvc -n ${target_ns} -o json | jq -c '.items | .[]')
+          pop_volume_name=""
+          for pvc in ${pvcs[@]}; do
+            # Volume populator prime PVC
+            if [ $(echo $pvc | jq -r '.metadata.name') = "$pvc_name" ]; then
+              pop_volume_name=$(echo $pvc | jq -r '.spec.volumeName')
```

Comment:
how do we know that this PVC was populated with a volume populator here (isn't it the same value we get in line 132)?

Reply:
we don't. but if it's using populator it will populate. otherwise it will remain empty string. it is the same value in line 132. but we "catch" it based on the pvc name in this particular case.

Reply:
ok, then we need to document this differently (instead of saying "Volume populator prime PVC"): when a disk is populated using a volume populator, the PVC's spec.volumeName is different from pvc_name and we'll have another PVC (prime PVC) that has the same value of spec.volumeName

Reply:
done